src/modules/dashboard/components/ColonyHomeActions/ColonyActionsItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import Icon from '~core/Icon';
+import Paragraph from '~core/Paragraph';
+import styles from './ColonyActionsItem.css';
+
+interface Props {
+  title: object;
+  description: object;
+  disabled?: boolean;
+}
+
+const MSG = defineMessages({
+  coming: {
+    id: 'dashboard.ColonyHomeActions.ColonyActionsItem.coming',
+    defaultMessage: 'Coming',
+  },
+});
+
+const ColonyActionsItem = ({ title, description, disabled }: Props) => {
+  return (
+    <div className={`${disabled ? styles.disabled : styles.content}`}>
+      <div>
+        <Paragraph className={styles.title}>
+          <Icon
+            appearance={{ size: 'medium' }}
+            name="caret-right"
+            title={title}
+          />
+          <FormattedMessage {...title} />
+          {disabled && (
+            <Paragraph className={styles.coming}>
```

Comment:
This is a DOM nesting violation, as a `<p>` element can't be nested under another `<p>`. See:

![Screenshot from 2020-11-11 11-13-25](https://user-images.githubusercontent.com/1193222/98792408-1d52b100-240f-11eb-8667-86915a8a5ce5.png)

Change this to a `span` and it will be all good